### DISCUSSION TITLE
Water and Sewerage Module Product Demo

### DISCUSSION THREAD

**Discussion by KrisP-Vassar** [egovernments/Digit-Core#48] [Water-Sanitation]
Dear Digit team,
 Please share/upload Water and Sewerage Module under the section of Training videos for Product Demo (@ https://urban.digit.org/products/training-and-demo/training-videos) . This will help our team to understand the functional usage and understand the workflows available in the Water and Sewerage module and customize as needed. 
 Thanks!
 Krishna P
 Vassar Labs India Pvt Ltd

**Comment by subhashini-egov** [comment]
Water and sewerage training videos are already available here --> [https://urban.digit.org/products/training-and-demo/training-videos]. Please see screenshot below:

<img width="860" alt="Screenshot 2022-09-19 at 2 49 25 PM" src="https://user-images.githubusercontent.com/107110442/190987118-21b766ab-1ebb-4bfd-9fb8-2fa8955a1537.png">
